coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -51,76 +53,111 @@ we get the output:
     flat_tax_on_salary:
         2016-01: 500
         2015-12: 400
-        ...
         2015-01: 400
         2014-12: 440
-        ...
         2014-01: 440
 ```
 
 [Read more about how to code parameters](./legislation_parameters.md#parameters-and-time).
 
-## Variable defined only for a specific time intervale
+## Formula evolving over time
+
+Some fiscal or benefit mechanism significantly evolve over time, with bigger changes than a simple parameter adjustement.
 
-As the legislation evolves, some fiscal or benefit mechanism appear and disapear. When creating a variable, you can specify the attribute `start_date` and `stop_date` to defines in which time intervale this variable make sense.
+For instance, let's assume that from the 1st of Jan. 2017, our previous `flat_tax_on_salary` will not be applied on the first `1000` earned by the person.
 
-When called outside of its definition intervale, a variable will **not** execute its formula and instead **return its default value**.
+We implement this rule by adding a new formula to our variable, and _dating_ it:
```

Comment:
This is the same system as the parameters evolution. Its there a way we could explain both in the same way ?

For variables : 
We implement this rule by adding a new formula to our variable, and _dating_ it:
for parameter :
We implement this rule by adding a new line to our parameter node, and _dating_ it: